test/Microsoft.ApplicationInsights.AspNetCore.Tests/SdkVersionTestUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Tests
+{
+    public class SdkVersionTestUtils
+    {
+        public static string GetExpectedSdkVersion()
+        {
+            string expectedSdkVersion = "aspnet5";
+
+#if NET451
+        expectedSdkVersion = expectedSdkVersion + "F";
```

Comment:
Why not call GetFrameworkType() instead of repeating the "F" and "C" logic and ifdefs?